src/datanode/src/heartbeat.rs

```diff
@@ -90,6 +92,10 @@ impl HeartbeatTask {
                     None
                 }
             } {
+                if let Some(msg) = res.mailbox_message.as_ref() {
+                    info!("Received mailbox message: {msg:?}, meta_client id: {client_id:?}");
```

Comment:
It's not big deal the `msg`s are too large in the logs, because they are not frequently sent from Metasrv. However, once sent, they are usually Important. So I think it's better to record all the fields here.

Reply:
For now the mailbox messages are not many, and the other communication logs are all debug level, I think it's not a necessity to separate logging now.